src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public Ball getRandomBall() {
+
+        Ball ball = new Ball();
```

Comment:
it's better to write code like this
 String randomColor = colorSupplier.getRandomColor();
int randomNumber = ThreadLocalRandom.current().nextInt(100);
return new Ball(randomColor, randomNumber);